### DISCUSSION TITLE
Serverless support

### DISCUSSION THREAD

**Discussion by ravirajb** [lunatic-solutions/lunatic#207] [General]
Looking to hear from the team on below points

1. Ways to deploy this as serverless
2. Http triggers 
3. Database support

**Comment by bkolobara** [comment]
Serverless is a bit vendor-specific. What platform (aws lambda, cloudflare functions, knative, ... ) are you targeting?

For example, today you can package lunatic into a Docker container and deploy on [Knative](https://knative.dev/docs/). Then you can just use a web framework like https://github.com/lunatic-solutions/submillisecond to build your app. But we don't have any bindings to aws lambda yet.

There are some libraries that provide bindings to popular databases: https://crates.io/crates/lunatic-redis https://crates.io/crates/lunatic-db.

We also have been working on our own platform, specifically tailored around lunatic's capabilities. Stay tuned, more information is coming soon :).

**Comment by ravirajb** [comment]
Hi,

Thanks for the quick response. Am more so looking for spin, but knative is fine / azure kubernetes (as mine is azure heavy for now)

That said, any thoughts on async capabilities within a container? I have a usecase of calling multiple api's and aggregating the response. Any thoughts on how to achieve this?
